framework/Models/Code.py

```diff
@@ -308,8 +312,11 @@ def createNewInput(self,currentInput,samplerType,**kwargs):
     if not os.path.exists(subDirectory):
       os.mkdir(subDirectory)
     for index in range(len(newInputSet)):
-      newInputSet[index].setPath(subDirectory)
-      shutil.copy(self.oriInputFiles[index].getAbsFile(),subDirectory)
+      subSubDirectory = os.path.join(subDirectory,newInputSet[index].subDirectory)
+      if newInputSet[index].subDirectory.strip() != "" and not os.path.exists(subSubDirectory):
+        os.mkdir(subSubDirectory)
```

Comment:
The reason why we do that (for now) is the fact that just at the "createNewInput" stage we know the "prefix" of the realization and so we can create the folder "workingDir/stepName/%prefix%"